lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -594,11 +597,16 @@ def write(chunk)
             unless record_json.nil?
               record = record_json
               is_json = true
+              # Extract LogEntry fields if they are present in the nested JSON.
```

Comment:
This comment is no longer accurate, right?

Reply:
Removed.